bot/code_review_bot/tasks/docupload.py

```diff
@@ -77,8 +77,18 @@ def build_notice(self, artifacts, revision):
         if trees is None:
             logger.warn("Missing trees.json")
 
-        doc_files = [file for file in revision.files if "docs" in file]
+        doc_files = [
+            file
+            for file in revision.files
+            if "docs" in file and file.endswith((".rst", ".md"))
+        ]
         nb_docs = len(doc_files)
+        if not nb_docs:
+            logger.info(
+                "Found no documentation file in revision, skipping comment creation"
+            )
+            return ""
```

Comment:
You should just use `return` here (I mean return `None`) and the calling code should behave accordingly, skipping publication

Reply:
`""` is returned [above](https://github.com/mozilla/code-review/pull/1460/files#diff-774da06547f30fc28ca6c29ce9a4c274e67c173fbad29f840dd69087644cec3dR72), we aren't completely skipping the comment creation (I guess the log is a little misleading), we just don't append anything, related to the documentation, at the end of the existing one